static/js/runtime-config-listeners/analytics.js

```diff
@@ -0,0 +1,13 @@
+import { canonicalizeBoolean } from '../utils';
+
+export default {
+  key: 'analyticsEventsEnabled',
+  callback: value => {
+    window.AnswersExperience.AnswersInitializedPromise
+      .then(() => {
+        const canonicalizedValue = canonicalizeBoolean(value);
+        ANSWERS.setAnalyticsOptIn(canonicalizedValue);
+      })
+      .catch(err => console.warn(err));
```

Comment:
if an error is caught, is the console.warn duplicated between the two runtime config listeners, since they both have this catch?

Reply:
ya, good catch (pun intended)

Reply:
I think it makes sense to remove the catches from here then